inc/Editor/BlockManagement/BlockRegistration.php

```diff
@@ -49,8 +49,20 @@ public static function register_helper_blocks(): void {
 		// Remote data pagination block - used to render pagination links for collections.
 		register_block_type( REMOTE_DATA_BLOCKS__PLUGIN_DIRECTORY . '/build/blocks/remote-data-pagination' );
 
-		// Remote data empty result block - used to render a message when no results are found.
-		register_block_type( REMOTE_DATA_BLOCKS__PLUGIN_DIRECTORY . '/build/blocks/remote-data-no-results' );
+		// Remote data error fallback block - used to render a message when results aren't available to load.
```

Comment:
Update comment to indicate that it registers both the block and its variation.

Reply:
Good point, done. I hadn't done that as the variation was happening in line here